### PR TITLE
[Docs] Add status/wontfix label

### PR DESCRIPTION
**Pull request by SaschaMann**
Motivation: Sometimes it turns out that an exercise idea doesn't work, or that a concept should not be taught. This status allows one to tell apart such issues/PRs from issues/PRs that have been closed due to being fixed or abandoned.

### CODE REVIEW

File: docs/maintainers/repository-structure.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ The following labels are used to categorize [issues in the v3 repository][github
 - [status/help-wanted][github-issues-status-help-wanted]: Extra attention is needed
 - [status/in-progress][github-issues-status-in-progress]: This issue is being worked on
 - [status/draft][github-issues-status-draft]: This issue or pull request is a draft, and not ready for review/merge.
+- [status/wontfix][github-issues-status-wontfix]: This issue or pull request is out of scope and will not be fixed.
```
Comment (ErikSchierboom) [inline docs/maintainers/repository-structure.md:55]: As "won't fix" is two words, should it be:

```suggestion
- [status/wont-fix][github-issues-status-wontfix]: This issue or pull request is out of scope and will not be fixed. 
```

Comment (SaschaMann) [reply docs/maintainers/repository-structure.md:55]: `wontfix` is the canonical spelling of this label/status in the context of bug trackers

**Comment by SaschaMann** [comment]
Thanks for the fast review!

**Comment by ErikSchierboom** [comment]
Sure!

**Pull request merged**